clang/lib/Sema/SemaChecking.cpp

```diff
@@ -11678,9 +11678,12 @@ static void AnalyzeImplicitConversions(
 
   // Propagate whether we are in a C++ list initialization expression.
   // If so, we do not issue warnings for implicit int-float conversion
-  // precision loss, because C++11 narrowing already handles it.
-  bool IsListInit = Item.IsListInit ||
-                    (isa<InitListExpr>(OrigE) && S.getLangOpts().CPlusPlus);
+  // precision loss, because C++11 narrowing already handles it. HLSL's
```

Comment:
The HLSL comment is completely different from the c++ narrowing comment. Feel like it should be on its own line  so as not to communicate that they are related.